### DISCUSSION TITLE
Default tasks need a root folder

### DISCUSSION THREAD

**Discussion by quachpas** [WeetHet/typst.zed#16] [Feature requests and discussion]
https://github.com/WeetHet/typst.zed/commit/8e07495fc8deeeec2296e9a43fe1a1a028898a3e#diff-814ee435ae0ab927e5a5d4aed8aac1357b68fa2ad53808c53800b1d46e2244c4R10

The tasks probably need a default root, i.e. `$ZED_WORKTREE_ROOT`.

**Comment by WeetHet** [comment]
I personally don't think they do. A lot of (my personal) typst usage is with single-file projects, which are not affected. You can always add tasks locally, so trying to guess which root works would probably only worsen the UX
